js/rpg_objects/Game_Interpreter.js

```diff
@@ -1730,3 +1739,60 @@ Game_Interpreter.prototype.command356 = function() {
 Game_Interpreter.prototype.pluginCommand = function(command, args) {
     // to be overridden by plugins
 };
+
+Game_Interpreter.requestImages = function(list){
+    list.forEach(function(command){
+        var params = command.parameters;
+        switch(command.code){
+            // Show Text
+            case 101:
+                if(params[0]) ImageManager.requestFace(params[0]);
+                break;
+
+            // Set Movement Route
+            case 205:
+                if(params[1]){
+                    params[1].list.forEach(function(command){
+                        var params = command.parameters;
+                        if(command.code === Game_Character.ROUTE_CHANGE_IMAGE){
+                            ImageManager.requestCharacter(params[0]);
+                        }
+                    });
+                }
+                break;
+
+            // Show Picture
+            case 231:
+                if(params[1]) ImageManager.requestPicture(params[1]);
+                break;
+
+            // Change Tileset
+            case 282:
+                var tileset = $dataTilesets[params[0]];
+                tileset.tilesetNames.forEach(function(tilesetName){
+                    if(tilesetName) ImageManager.requestTileset(tilesetName);
+                });
+                break;
+
+            // Change Actor Images
+            case 322:
+                if(params[1]) ImageManager.requestCharacter(params[1]);
+                if(params[3]) ImageManager.requestFace(params[3]);
+                if(params[5]) ImageManager.requestEnemy(params[5]);
+                break;
+
+            // Change Vehicle Image
+            case 323:
+                var vehicle = $gameMap.vehicle(params[0]);
+                if(vehicle && params[1]){
+                    ImageManager.requestCharacter(params[1]);
+                }
+                break;
+
+            // Show Battle Animation
+            case 337:
+                if(params[1]) ImageManager.requestAnimation(params[1]);
+                break;
+        }
```

Comment:
Can you add request below?

```
case 129 (Change Party Member)
case 212 (Show Animation)
case 283 (Change Battle Back) <- heavy?
case 284 (Change Parallax) <- heavy?
battle action actors and enemies common use. <- It's big coding, so it may be good to separate PR
```